ROMFS/px4fmu_common/init.d/rc.gnd_apps

```diff
@@ -0,0 +1,24 @@
+#!nsh
+#
+# Standard apps for fixed wing
+#
+
+#
+# Start the attitude and position estimator
+#
+ekf2 start
+#attitude_estimator_q start
+#local_position_estimator start
+
+
+#
+# Start attitude controller
+#
+gnd_att_control start
+gnd_pos_control start
+
+
+#
+# Start Land Detector
+#
+land_detector start rover
```

Comment:
I ask myself if a land detection is required for an UGV. Unless we are considering jumps, as in the case of RC racing cars, which I don't believe it is the case and wouldn't interfere with the control logic.